src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -522,24 +534,39 @@ public double getGripperPivotAbs() {
         return m_absGripperPivotEncoder.get();
     }
 
-    public void setOveride(boolean overide) {
-        m_brake = overide;
+    public void setOverride(boolean override) {
+        m_overrideCalls += override ? 1 : -1;
     }
 
     public boolean getOveride() {
-        return m_brake;
+        return m_overrideCalls > 0;
     }
 
     public void toggleBreak() {
-        m_breakSolenoid.toggle();
-    }
-
-    public void setBreak(boolean brake) {
-        m_breakSolenoid.set(brake);
+        m_brakeSolenoid.toggle();
+    }
+
+    public void openBrake(boolean brake) {
```

Comment:
I changed the name to (hopefully) make it a bit more clear that 'true' values cause the brake to open.